packages/nimble-blazor/package.json

```diff
@@ -1,49 +1,40 @@
 {
-  "name": "@ni/nimble-blazor",
-  "version": "16.0.1",
-  "description": "Blazor components for the NI Nimble Design System",
+  "name": "@ni-private/blazor-workspace",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Workspace for Blazor projects",
   "scripts": {
     "build": "npm run generate-icons && npm run generate-hybrid && npm run build:release && npm run build:client",
     "build:release": "dotnet build -c Release /p:TreatWarningsAsErrors=true /warnaserror",
     "build:client": "dotnet publish -p:BlazorEnableCompression=false -c Release Examples/Demo.Client --output dist/blazor-client-app",
     "generate-icons": "npm run generate-icons:bundle && npm run generate-icons:run",
     "generate-icons:bundle": "rollup --bundleConfigAsCjs --config build/generate-icons/rollup.config.js",
     "generate-icons:run": "node build/generate-icons/dist/index.js",
-    "generate-hybrid": "rollup --bundleConfigAsCjs --config build/generate-hybrid/rollup.config.js",
+    "generate-hybrid": "npm run generate-hybrid:nimble && npm run generate-hybrid:spright",
+    "generate-hybrid:nimble": "rollup --bundleConfigAsCjs --config build/generate-hybrid/rollup.nimble.config.js",
+    "generate-hybrid:spright": "rollup --bundleConfigAsCjs --config build/generate-hybrid/rollup.spright.config.js",
     "lint": "npm run lint:cs && npm run lint:js",
     "lint:cs": "dotnet format --verify-no-changes",
     "lint:js": "eslint .",
     "format": "npm run format:cs && npm run format:js",
     "format:cs": "dotnet format",
     "format:js": "eslint . --fix",
     "test": "dotnet test -c Release",
-    "pack": "cross-env-shell dotnet pack NimbleBlazor -c Release -p:PackageVersion=$npm_package_version --output dist",
-    "invoke-publish": "npm run invoke-publish:clean-nuget && npm run pack && npm run invoke-publish:nuget && npm run invoke-publish:npm -- ",
-    "invoke-publish:clean-nuget": "rimraf --glob \"dist/*.nupkg\"",
-    "invoke-publish:nuget": "cross-env-shell dotnet nuget push \"dist/*.nupkg\" -k $NUGET_SECRET_TOKEN -s \"https://api.nuget.org/v3/index.json\"",
-    "invoke-publish:npm": "echo \"noop command to swallow npm args\"",
-    "copy-resources": "node build/copyNimbleResources.js"
+    "copy-resources": "npm run copy-resources:nimble && npm run copy-resources:spright",
```

Comment:
Each package runs the commands separately, don't need this anymore for calling both